src/profile.c

```diff
@@ -239,6 +239,49 @@ static void CheckLeaveFunctionsAfterLongjmp(void)
     }
 }
 
+static Obj JsonEscapeString(Obj param)
+{
+    Int lenString = LEN_LIST(param);
+
+    // Massively over-long string
+    Obj     copy = NEW_STRING(lenString * 2 + 2);
+    UChar * in = CHARS_STRING(param);
+    UChar * base = CHARS_STRING(copy);
+    UChar * out = base;
+
+    for (Int i = 0; i <= lenString; ++i) {
```

Comment:
That's the wrong bounds. Perhaps you did it to "cleverly" set the zero terminator, but I'd argue that'd be too clever for its own good. Plus it is not necessary as the newly allocated string is guaranteed to be zeroed.
```suggestion
    for (Int i = 0; i < lenString; ++i) {
```